js/ui/radio_group/radio_group.js

```diff
@@ -278,6 +280,8 @@ class RadioGroup extends Editor {
             selectedItemKeys: [this.option('value')],
             tabIndex: this.option('tabIndex')
         });
+
+        this.isRadiosRendered.resolve();
```

Comment:
Do we need to move this to `_fireContentReadyAction`? I suggest writing a test with an asynchronous data source to ensure this.

Reply:
I have added the test